### PR TITLE
[PD 2977] remove access

### PR DESCRIPTION
**Pull request by mwarnsleyDE**
## What's this PR do?
This PR removes all references to Classic Reporting in Reports and strictly goes to dynamic reporting 100% of the time.

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start at the reports home page found in the navigation bar of secure.my.jobs

## How should this be manually tested?
Pull Down the Repository and go to the navigation bar clicking on Reports. Reports should not ask for or allow for you to go to Classic Reporting and should go straight to dynamic reporting view.

## Any background context you want to provide?
All Laughs from the Python guys are welcomed as far as the views.py file that was edited by the FRONT END Developer lol.

### CODE REVIEW

File: gulp/src/reporting/components/ReportList.jsx [hunk 1]
```diff
@@ -138,12 +138,6 @@ class ReportList extends Component {
               {reportLinks}
             </ul>
           </div>
-          <h2>Reporting Version</h2>
-          <button
-            className="button primary wide"
-            onClick={() => this.handleSwitchVersions()}>
```
Comment (tdphillips) [inline gulp/src/reporting/components/ReportList.jsx]: Should handleSwitchVersions be removed as well?

Comment (mwarnsleyDE) [inline gulp/src/reporting/components/ReportList.jsx]: @tdphillips I just took a look at this and removed it and I believe removing it doesn't give you the chance to create a new report unless I'm not doing it correctly

Comment (mwarnsleyDE) [inline gulp/src/reporting/components/ReportList.jsx]: nvm troy

Comment (mwarnsleyDE) [inline gulp/src/reporting/components/ReportList.jsx]: I was looking in the wrong spot that's my fault

**Comment by tdphillips** [comment]
I suspect there's going to be lots of test fixes. We've probably got a bunch still looking at old reporting.

Discussion point that we should probably talk about:

Remove the "overview" view in its entirety and modify myreports/urls.py to point view/overview to the dynamicoverview view.
After that, is a RedirectView from view/dynamicoverview to view/overview necessary?

**Pull request merged**